[2024-10-29T09:08:41.877+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-29T09:08:41.983+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2024-10-28T00:00:00+00:00 [queued]>
[2024-10-29T09:08:42.003+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2024-10-28T00:00:00+00:00 [queued]>
[2024-10-29T09:08:42.004+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-29T09:08:42.092+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2024-10-28 00:00:00+00:00
[2024-10-29T09:08:42.115+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=204) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-29T09:08:42.118+0000] {standard_task_runner.py:72} INFO - Started process 205 to run task
[2024-10-29T09:08:42.118+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'second_task', 'scheduled__2024-10-28T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpie6qobk2']
[2024-10-29T09:08:42.122+0000] {standard_task_runner.py:105} INFO - Job 90: Subtask second_task
[2024-10-29T09:08:42.663+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v3.second_task scheduled__2024-10-28T00:00:00+00:00 [running]> on host 2d730dc1e71d
[2024-10-29T09:08:42.995+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-28T00:00:00+00:00'
[2024-10-29T09:08:42.995+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-29T09:08:43.039+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-29T09:08:43.040+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am the second task after task 1']
[2024-10-29T09:08:43.053+0000] {subprocess.py:86} INFO - Output:
[2024-10-29T09:08:43.056+0000] {subprocess.py:93} INFO - hey, I am the second task after task 1
[2024-10-29T09:08:43.057+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-29T09:08:43.120+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-29T09:08:43.120+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=second_task, run_id=scheduled__2024-10-28T00:00:00+00:00, execution_date=20241028T000000, start_date=20241029T090841, end_date=20241029T090843
[2024-10-29T09:08:43.203+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-29T09:08:43.240+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-29T09:08:43.270+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
